textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/ListField.java

```diff
@@ -1,20 +1,33 @@
-package edu.uci.ics.textdb.common.field;
-
-import java.util.List;
-
-import edu.uci.ics.textdb.api.common.IField;
-
-public class ListField<T> implements IField{
-
-    private List<T> list;
-    
-    public ListField(List<T> list){
-        this.list = list;
-    }
-    
-    @Override
-    public List<T> getValue() {
-        return list;
-    }
-
-}
+package edu.uci.ics.textdb.common.field;
+
+import edu.uci.ics.textdb.api.common.IField;
+
+import java.util.List;
+
+public class ListField<T> implements IField {
```

Comment:
This will be used, to return an `ITuple` that has a list of strings, that is the list of rewritten query strings that our operator sends.